index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Website</title>
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+</head>
+<body>
+<nav class="bg-indigo-800 flex flex-row text-white h-1/6 justify-center space-x-6 ">
+    <a class="hover:text-gray-900 shadow" href="#">Home</a>
+    <a class="hover:text-gray-900" href="contact.html">Contact</a>
```

Comment:
Change the `href` to `#` (unless you have a contact.html file that you're writing)

Reply:
thanks. I have decided to  replace with href="mailto:tzvekare@gmail.com? subject= Hello Tariro"> as per your illustration earlier today